frontend/src/pages/AdminPage/AdminPage.js

```diff
@@ -37,22 +39,24 @@ function AdminPage() {
   const { isShowing, toggle } = useModal();
 
   return (
-    <div>
-      <Header type="admin" />
-      <h1 className="main-title">Ticket Types</h1>
-      {fakeData.map(adminTicket => (
-        <Ticket key={adminTicket.id} {...adminTicket}>
-          <Button onClick={toggle}>DELETE</Button>
-        </Ticket>
-      ))}
-      <Modal hide={toggle} isShowing={isShowing}>
-        <Button onClick={toggle}>OK</Button>
-        <Button buttonStyle={'btn--danger--solid--btn'} onClick={toggle}>
-          Cancel
-        </Button>
-        Are you sure?
-      </Modal>
-    </div>
+    <Provider store={store}>
+      <div>
+        <Header type="admin" />
+        <h1 className="main-title">Ticket Types</h1>
+        {fakeData.map(adminTicket => (
+          <Ticket key={adminTicket.id} {...adminTicket}>
+            <Button onClick={toggle}>DELETE</Button>
+          </Ticket>
+        ))}
+        <Modal hide={toggle} isShowing={isShowing}>
+          <Button onClick={toggle}>OK</Button>
+          <Button buttonStyle={'btn--danger--solid--btn'} onClick={toggle}>
+            Cancel
+          </Button>
+          Are you sure?
+        </Modal>
+      </div>
+    </Provider>
```

Comment:
I don't think you need provider here because the entire app is wrapped in it

Reply:
theoretically yes, practically tests are failed without it. I'll try to add it just in the test file later, maybe it can work

Reply:
Then you should definitely remove it if only the tests fail, test like you tested something that was wrapped in a <Link/> where you mocked the router.

https://github.com/reduxjs/redux-mock-store